tests/installation/bootloader_uefi.pm

```diff
@@ -49,12 +49,14 @@ use version_utils qw(is_jeos is_microos is_sle);
 sub run {
     my ($self) = @_;
 
-    # Enable boot menu for x86_64 uefi workaround, see bsc#1180080 for details
+    # Enabled boot menu for x86_64 uefi workaround because of bsc#1180080, however,
+    # for migration tests, before migration, we need to boot the harddisk and fully
+    # pach it which is our testing requirement. So we keep this logic to boot the
+    # harddisk directory instead of cdrom boot menu entry
     # Case setting also need BOOT_MENU=1 to support it
     if (is_sle && get_required_var('FLAVOR') =~ /Migration/ && is_x86_64) {
         # Skip workaround on specific scenaio which call this module after migration
         if (!check_screen('bootloader-grub2', 0, no_wait => 1)) {
-            record_soft_failure 'bsc#1180080';
```

Comment:
If it is not a workaround, I think we should not  have ' workaround because of bsc#1180080,' comment.

Reply:
Agree with Yutao!

Reply:
Suggest change: 
In migration tests, we set cdrom as boot index=0, however.....

Reply:
Thanks, updated the comment, please help to review.